deploy/operator.yaml

```diff
@@ -19,13 +19,11 @@ spec:
           # Replace this with the built image name
           image: REPLACE_IMAGE
           command:
-          - compute-node-operator
+            - compute-node-operator
```

Comment:
no need for this indentation, right?

Reply:
yeah was done automatically by my editor, reverted it